src/views/blog-post/blog-post-layout/blog-post-layout.module.scss

```diff
@@ -87,3 +87,18 @@
 		opacity: 1;
 	}
 }
+
+.mobileMenu {
```

Comment:
I've noticed that, while the menu is expanded, it is still possible to scroll the page behind it by swiping on mobile - and this also changes the selected heading while the ToC is open (even though you can't see the content behind it).

Applying `overflow: hidden` to the `<body>` should prevent this - and that _won't_ cause layout shift in this case because we have `scrollbar-gutter: stable` in our CSS :)

(the search page / filter modals previously ran into this same issue)

Reply:
https://github.com/josephmfaulkner/playfulprogramming/commit/7befaca0334e58af2f9cfe39acc732ad02308a2d